src/System.Management.Automation/engine/parser/Position.cs

```diff
@@ -230,20 +230,32 @@ internal static string VerboseMessage(IScriptExtent position)
                 }
 
                 if (needsPrefixDots)
-                    sb.Append("... ");
+                {
+                    sb.Append("\u2026 ");
+                }
+
                 sb.Append(sourceLine);
+
                 if (needsSuffixDots)
-                    sb.Append(" ...");
+                {
+                    sb.Append(" \u2026");
+                }
+
                 sb.Append(Environment.NewLine);
                 sb.Append("+ ");
-                sb.Append(' ', spacesBeforeError + (needsPrefixDots ? 4 : 0));
+                sb.Append(' ', spacesBeforeError + (needsPrefixDots ? 2 : 0));
```

Comment:
Minor comment: there is a cache for space strings, can we use them here?

Reply:
I think this existing code is ok, only change is the size as the Unicode ellipsis is 1 char plus a space vs 3 dots and a space.